### DISCUSSION TITLE
[QUESTION] LEAVEONEND

### DISCUSSION THREAD

**Discussion by Samiul30** [Androz2091/discord-player#620] [Q&A]
**Question**
How can I set leaveOnEnd per server?

**Comment by twlite** [comment]
Not possible but we have planned this for v5

**Comment by Samiul30** [comment]
> Not possible but we have planned this for v5

Thank you. Hope to get it as soon as possible.

**Comment by Samiul30** [comment]
Sorry for another question. I have set leaveOnEnd false in client options and do some staff with stop.js where if the mongo has the guild id then it will not disconnect otherwise it will disconnect from the VC by the code voice.channel.leave() though I have set leaveOnStop: false. Will there will be any err in further?
